### DISCUSSION TITLE
How do I change my email address?

### DISCUSSION THREAD

**Discussion by ninjamar** [supabase/supabase#6337] [Questions] [answered]
How do I change my email address? (app.supabase.io > account > preferences)

**Comment by GaryAustin1** [comment, marked as answer]
The only way I have seen suggested is to contact support.  I’ve never seen a reponse if it was able to be done. If you go that route please respond back if they were able to change it, although I assume they have to be able to that.

**Reply by ninjamar** [reply]
I contacted support who changed it

**Comment by knmts** [comment]
_For those who see this page in the future_

I contacted support and its response is below. As of April 2023.
> Right now, change email address is not possible, the best approach to this is to migrate your projects to your other account.
> [https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#migrate-your-project](https://supabase.com/docs/guides/database/migrating-between-projects)

**Comment by huksley** [comment]
What happens if I change my email in github and login again? Now it does not update email automatically. Shows empty field and HTTP 409 conflict error in console.

![image (1)](https://user-images.githubusercontent.com/1594349/236817780-4acaa602-a2cf-4aca-a92b-a0dfaa9441d1.png)
